src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BaseRecalOutput.java

```diff
@@ -54,4 +54,5 @@ public StandardCovariateList getCovariates() {
     }
 
 
+
```

Comment:
It's not a big deal at all.  It just makes the change set easier to understand though if there aren't random extra changes.